src/StorageMover/README.md

```diff
@@ -0,0 +1,166 @@
+<!-- region Generated -->
+# Az.StorageMover
+This directory contains the PowerShell module for the StorageMover service.
+
+---
+## Status
+[![Az.StorageMover](https://img.shields.io/powershellgallery/v/Az.StorageMover.svg?style=flat-square&label=Az.StorageMover "Az.StorageMover")](https://www.powershellgallery.com/packages/Az.StorageMover/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.StorageMover`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+# Please specify the commit id that includes your features to make sure generated codes stable.
+brach: 913f1e8aa0bfc773c5f236f03d56f379d9d3d012
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/storagemover/resource-manager/Microsoft.StorageMover/preview/2022-07-01-preview/storagemover.json
+
+# For new RP, the version is 0.1.0
+module-version: 0.1.0
+# Normally, title is the service name
+title: StorageMover
+subject-prefix: $(service-name)
+nested-object-to-string: true
+identity-correction-for-post: true 
+
+directive:
+  - where:
+      verb: Set
+    remove: true
+  - where:
+      model-name: StorageMover
+    set:
+      suppress-format: true
+  - where:
+      model-name: JobDefinition
+    set:
+      suppress-format: true
```

Comment:
`model-name` is actually regular expression, so you can group the same directives together like:
```suggestion
  - where:
      model-name: StorageMover|JobDefinition
    set:
      suppress-format: true
```
You don't HAVE to group them. It's just a matter of coding style

Reply:
Thanks for the suggestion. Yes this looks much cleaner than listing them out one by one. Fixed.